test/api_integration/apis/index_patterns/runtime_fields_crud/put_runtime_field/errors.ts

```diff
@@ -40,30 +40,5 @@ export default function ({ getService }: FtrProviderContext) {
 
       expect(response.status).to.be(404);
     });
-
-    it('returns error on non-runtime field update attempt', async () => {
```

Comment:
No need of this test as the lookup is in the runtimeFieldMap and not the field list, so there is no way to edit a non-runtime field through API.